tests/regtestenv.rs

```diff
@@ -60,7 +66,7 @@ impl RegTestEnv {
             wallet.sync(&blockchain, SyncOptions::default()).unwrap();
             let balance = wallet.get_balance().unwrap();
             assert!(
-                balance.confirmed == 5_000_000_000,
+                balance.confirmed == 50_000_000_000,
                 "balance of wallet {} is {} but should be 5'000'000'000",
```

Comment:
⛏️  Same comment as above plus it should be 50M and not 5M. Why did the balance change?
```suggestion
                "balance of wallet {} is {} but should be 50_000_000_000",
```